frontend/src/components/utils/sidebar/SidebarInputSearch.tsx

```diff
@@ -1,6 +1,19 @@
 import React from "react";
 import { DebounceInput } from "react-debounce-input";
 
+/**
+ * Debounced input field for the sidebar
+ * @param setParams the useState function to update the `search` param
```

Comment:
It does not _have_ to be a function from `useState`, maybe use a more generic phrasing, like a "callback"